### PR TITLE
fix: update release tag name on pre-releases

### PR DESCRIPTION
**Pull request by sigilioso**
Updates the pre-release tag name when the pre-release success or fails

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -71,6 +71,16 @@ jobs:
...
           make upload_manifests RELEASE_VERSION=$NRI_VERSION NRI_ECS_IMAGE_TAG=$DOCKER_IMAGE_TAG
 
+      - if: ${{ github.event.release.prerelease }}
+        uses: actions/checkout@v4
+      - if: ${{ github.event.release.prerelease }}
+        name: Update title for successful pre-release
+        env:
+          GH_TOKEN: "${{ secrets.COREINT_BOT_TOKEN }}"
+        run: |
+          gh release edit ${{ github.event.release.tag_name  }} --title "${{ github.event.release.tag_name  }}"
```
Comment (paologallinaharbur) [inline .github/workflows/release.yml:81]: I've just realised I've missed this step in the discovery one 😵‍💫

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -82,3 +92,15 @@ jobs:
           slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
           slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
           slack-text: "❌ `${{ env.ORIGINAL_REPO_NAME }}`: [image release failed](${{ github.server_url }}/${{ env.ORIGINAL_REPO_NAME }}/actions/runs/${{ github.run_id }})."
+
+  update-title-on-failure:
```
[the remark this answers is not in the document]
Comment (sigilioso) [reply .github/workflows/release.yml:96]: Yes, it makes total sense

[the remark this answers is not in the document]
Comment (sigilioso) [reply .github/workflows/release.yml:96]: I've just updated it here: [e2ecfab](https://github.com/newrelic/nri-ecs/pull/147/commits/e2ecfab860ae4c68b10e5d1fc0523aad1513c956).

I also considered using a `GITHUB_ENV` variable to compose the tittle but, I'm not sure if that would have been more complex. Let me know your thoughts.

[the remark this answers is not in the document]
Comment (paologallinaharbur) [reply .github/workflows/release.yml:96]: I think it is clear enough like this, it is just few lines below, therefore I think it is ok without jumping from one side to the other

**Review by paologallinaharbur — APPROVED** [review]

**Pull request merged**